cpp/ql/src/semmle/code/cpp/Class.qll

```diff
@@ -828,7 +853,18 @@ class LocalClass extends Class {
 }
 
 /**
- * A nested class [4140 9.7].
+ * A nested class [4140 9.7].  For example the struct `PairT` in the following
```

Comment:
I think it would be more helpful to explain what a nested class is than to refer to the C++14 standard. Maybe, make it similar to the doc for `LocalClass`: "A class that is directly enclosed by another class."

I assume that the word "class" in this file means "class, struct, or union", but that may not be clear to a reader who is only viewing the QLDoc for a single element in this file. So should we fully write out "class, struct, or union" everywhere? Or should we use `Class`, in backticks, to refer to the QLDoc of `Class`? Another option is to consistently use `struct` or `union` instead of `class` in examples.

Reply:
Done the first part.

Regarding the second, I think my preference would be the somewhat wordy but super-clear "class, struct or union".  Does anybody else have an opinion on this?

Reply:
Fixed the second part, using "class, struct or union" in most places but "class/struct" (copying an existing phrasing) where I'm fairly sure that "union" doesn't apply.